_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,473 +856,2421 @@ If you need to brush up on how the IaC Library works, read the
+navigate in the AWS web console to `My Account` and then to the `Personal Information` page. There you should be able
+to click on `Configure Security Challenge Questions` and add your questions.
+
+Enable MFA::
+Make sure to
+https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa[enable MFA for your root user].
+Feel free to use a virtual or hardware MFA device, whichever is more straightforward or required by your company, as either one
+dramatically improves the security of your root user. It is up to your discretion to decide which option is suitable
+for your use case. The CIS benchmark recommends using a dedicated or company-owned device for MFA and not a personal
+one. This applies to both virtual and hardware devices.
+
+Disable access keys::
+Make sure to
+https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_delete-key[delete the root user's access keys],
+so that the only way to login as the root user is via the web console, where MFA is required.
+
+Don't use the root user again::
+In the next section, you will create an IAM user in the root account with admin permissions. Once you've created that
+IAM user, you should do everything as that IAM user, and more or less never touch the root user account again.
+The only time you'll need it is for account recovery situations (e.g., you accidentally deleted the IAM user or lost
+your credentials) or for the
+https://docs.aws.amazon.com/general/latest/gr/aws_tasks-that-require-root.html[small number of tasks that require root user credentials].
+
+
+[[create_iam_user_in_root]]
+=== Create an IAM user in the root account
+
+As the last action you do as the root user, you MUST create an IAM user. This is not only a better practice from a
+security standpoint, but also, the `account-baseline-xxx` modules we will use below assume IAM roles, which does not
+work with a root user. Later on, we'll create and manage all IAM users as code, but you should create this very first
+IAM user manually by
+https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console[following these instructions]:
+
+. Enter a username for your IAM user.
+. Select both "programmatic access" and "AWS Management Console access."
+. On the next page, click "Attach existing policies to user directly" and attach the `AdministratorAccess` policy.
+. Click next a few more times to create the IAM user.
+. In a secrets manager, save the IAM sign-in URL, your IAM user's username, the password, and your Access Keys.
+
+[[lock_down_iam_users]]
+=== Lock down the root account IAM users
+
+Although IAM users don't have the same powers as a root user, having an IAM user account compromised can still be a
+huge problem for your company (especially if that IAM user had admin permissions), so it's still critical to lock down
+IAM user accounts as much as possible:
+
+Use a secrets manager::
```

Comment:
This section seems duplicated, just a NIT for we to reduce the amount of text in this article

Reply:
@marinalimeira Overall I agree with you, but as it's only 3 steps I'd rather leave them "as-is" to avoid confusion. Otherwise for "Lock down the root account IAM users", we'd have to say follow the recommendations in "Lock down the root user", but ignore all of them except for:

* Use a secrets manager
* Use a strong, generated password
* Enable MFA

So it ends up getting slightly verbose anyway.

Reply:
@ina-stoyanova @marinalimeira resolving by leaving it in for now.